password-auth/Cargo.toml

```diff
@@ -30,7 +30,7 @@ default = ["argon2", "std"]
 std = []
 
 [target.'cfg(target_arch = "wasm32")'.dependencies]
-getrandom = { version = "0.2", features = ["js"] }
+getrandom = { version = "0.3", features = ["wasm_js"] }
```

Comment:
Note that you enable this feature unconditionally for all downstream users. It may be better to just not test RNG-dependent features on WASM.

Reply:
But I'm not enabling it for all target am I?

Reply:
You are enabling it for all WASM targets, including stuff like WASI. The `wasm_js` feature is not intended to be enabled in libraries. At the very least you should do it as a `dev-dependency`.

Reply:
I guess since you simply port the old version which (incorrectly) enables the `js` feature, we can leave this change for a later PR.

Reply:
I changed it to have the `wasm_js` feature and this is disabled by default, consumer may enable it (and I'm enabling it in tests)